app/models/author.model.js

```diff
@@ -0,0 +1,17 @@
+module.exports = (sequelize, Sequelize) => {
+  const Author = sequelize.define("author", {
+    firstName: {
+      type: Sequelize.STRING,
+      allowNull: false,
```

Comment:
Shouldn't firstName also be nullable? i.e., with the edit below
```suggestion
      allowNull: true,
```

Reply:
Wrong work item!